### ISSUE TITLE
op11_no06: sf below staff and f in middle of staff

### ISSUE THREAD

**Issue by bel28kent** [bel28kent/Mysterium#57]
<img width="747" alt="Screen Shot 2024-05-05 at 5 57 40 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/0c021b89-ce11-4a7f-9106-3176c4dff1a0">

```tsv
!!!COM: Scriabin, Alexander
!!!OTL@@FR: Prélude en si mineur
!!!OTL@@EN: Prelude in B Minor
!!!OPR@@FR: 24 Préludes
!!!OPR@@EN: 24 Preludes
!!!OPS: 11
!!!ONM: 6
!!!ODT: 1889
!!!AGN: Prelude
**kern	**kern	**dynam
*clefF4	*clefF4	*
*k[f#c#]	*k[f#c#]	*
*b:	*b:	*
*M2/4	*M2/4	*
=19	=19	=19
[4EE-X [4E-X	8r	sf f
*	*clefG2	*
.	[8F#	.
8EE-L] 8E-]	4F#] 4cn 4f#	.
8DDJ 8D	.	.
!!LO:LB:g=original
=	=	=
*-	*-	*-
!!!system-decoration: {(*)}
!!!filter: flipper -a
!!!RDF**dynam: > = dim.
!!!ENC: Bryan Jacob Bell
!!!END: November 2021 to April 2022
!!!PED: Günter Philipp
!!!PTL: Ausgewählte Klavierwerke (Edition Peters)
```

Currently encoded with both markings in the same token. I tried splitting the `**dynam` spine, but I can't get `!LO:DY:Y` to move downwards.

**Comment by craigsapp** [comment]
This can be handled by a special case, where `z` in `**kern` data represents `sf`:

<img width="796" alt="Screenshot 2024-05-05 at 3 11 56 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/eaf8083a-b5a2-42e6-b2f6-fbc7f4fa01b7">

[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCSoqa2VybgkqKmR5bmFtCipzdGFmZjIJKnN0YWZmMQkqc3RhZmYxLzIKKmNsZWZGNAkqY2xlZkcyCSoKPTEJPTEJPTEKNEVFLXpcIDRFLXpcCTRyCWYKKi0JKi0JKi0K)]

----------------

To otherwise encode two dynamics at the same time, use a spine split:

<img width="1024" alt="Screenshot 2024-05-05 at 3 16 54 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/cea90d40-3683-443a-aaed-c0a2a0fceb7f">

[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCSoqa2VybgkqKmR5bmFtCipzdGFmZjIJKnN0YWZmMQkqc3RhZmYxLzIKKmNsZWZGNAkqY2xlZkcyCSoKPTEJPTEJPTEKKgkqCSpeCiEJIQkhTE86RFk6Yj0yCSFMTzpEWTpjCjRFRS0gNEUtCTRyCXNmCWYKKgkqCSp2CSp2CiotCSotCSotCg==)]


But the first example is better in this case (`sf` is an instantaneous dynamic which functions as an articulation, so best to use `z` in most cases, and I usually put in `**dyanam` when it applies to both staves and is centered in the middle of the grand staff.

**Comment by bel28kent** [comment]
Fixed in 7c50593

**Comment by bel28kent** [comment]
A related question: How would you move a cresc or decresc wedge below the staff?

<img width="390" alt="Screen Shot 2024-05-05 at 6 57 54 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/d4cfdbca-77a3-4329-bd72-a13d73556e2b">

```tsv
!!!COM: Scriabin, Alexander
!!!OTL@@FR: Prélude en la majeur
!!!OTL@@EN: Prelude in A Major
!!!OPR@@FR: 24 Préludes
!!!OPR@@EN: 24 Preludes
!!!OPS: 11
!!!ONM: 7
!!!ODT: 1895
!!!AGN: Prelude
**kern	**kern	**dynam
*clefF4	*clefG2	*
*k[f#c#g#]	*k[f#c#g#]	*
*A:	*A:	*
*M6/8	*M6/8	*
=13	=13	=13
*	*^	*
4FF# 4F#	16BLL	8aL 8b	.
.	16e	.	.
.	16cn	8a 8ccn	.
.	16e	.	.
!	!	!	!LO:DY:b=2
8BBB 8BB	16c	[8aJ [8cc	<
!	!	!	!LO:DY:b=2
.	16d#XJJ	.	(
!	!	!	!LO:DY:b=2
8GGnL 8Gn	16cLL	4a] 4cc]	[ >
!	!	!	!LO:DY:b=2
.	16e	.	)
!	!	!	!LO:DY:b=2
8FF# 8F#	16c	.	]
.	16d#	.	.
8AAAJ 8AA	16c	8a 8cc	.
.	16eJJ	.	.
*	*v	*v	*
=	=	=
*-	*-	*-
!!!system-decoration: {(*)}
!!!filter: flipper -a
!!!RDF**dynam: > = dim.
!!!ENC: Bryan Jacob Bell
!!!END: November 2021 to April 2022
!!!PED: Günter Philipp
!!!PTL: Ausgewählte Klavierwerke (Edition Peters)
```

**Comment by craigsapp** [comment]
<img width="907" alt="Screenshot 2024-05-05 at 6 16 06 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/f07d2653-eb28-42b3-ad57-4dd6658acffd">


[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCSoqa2VybgkqKmR5bmFtCipzdGFmZjIJKnN0YWZmMQkqc3RhZmYxLzIKKmNsZWZGNAkqY2xlZkcyCSoKIQkhCSFMTzpIUDpiPTIKNEQJNEIJPAohCSEJIUxPOkhQOmI9Mgo0RAk0QglbID4KNEQJNEIJXQo9CT0JPQoqLQkqLQkqLQo=)]


position `b=2` (below the second staff of the part).

**Comment by bel28kent** [comment]
Fixed in 6f7a2ef
